CHANGES.rst

```diff
@@ -182,6 +182,8 @@ Unreleased
     previous item. :issue:`1353`
 -   The ``Path`` param type can be passed ``path_type=pathlib.Path`` to
     return a path object instead of a string. :issue:`405`
+-   TypeError is raised when parameter with ``multiple=True`` or
```

Comment:
`TypeError` is a class name, it should be code formatted with double backticks.

Reply:
Missed that one. Will fix it.